packages/marko-web-theme-default/i18n.js

```diff
@@ -0,0 +1,11 @@
+const translations = require('./translations.js');
+
+export default function i18n(lang, fieldLabelKey) {
```

Comment:
This should use CommonJS modules (e.g. `module.exports`) not ESM (`export default`)